nextstrain/cli/__main__.py

```diff
@@ -13,30 +13,24 @@ def main():
     # rest of the codebase.  Avoids needing to instruct folks to set
     # PYTHONIOENCODING=UTF-8 or use Python's UTF-8 mode (-X utf8 or
     # PYTHONUTF8=1).
-    sys.stdout = reconfigure_stdio(sys.stdout) # type: ignore[arg-type]
-    sys.stderr = reconfigure_stdio(sys.stderr) # type: ignore[arg-type]
+    reconfigure_stdio(sys.stdout) # type: ignore[arg-type]
+    reconfigure_stdio(sys.stderr) # type: ignore[arg-type]
 
     return cli.run( argv[1:] )
 
 
-def reconfigure_stdio(stdio: TextIOWrapper) -> TextIOWrapper:
+def reconfigure_stdio(stdio: TextIOWrapper):
     """
     Reconfigure *stdio* to match the assumptions of this codebase.
 
     Suitable only for output streams (e.g. stdout, stderr), as reconfiguring an
     input stream is more complicated.
     """
-    # XXX TODO: When we drop Python 3.6 support, most of this function can be
-    # replaced by stdio.reconfigure().
-    #   -trs, 6 June 2022
-
     # Flush any pending output under old configuration.
     stdio.flush()
```

Comment:
This can go. From the `.reconfigure()`'s methods docs:

> This method does an implicit stream flush before setting the new parameters.

Similarly, the explicit setting of `line_buffering` below can go away, as it will be preserved by using `.reconfigure()`.

Reply:
Thanks for the tips, updated in 58104afa0c253fad48185437ae93c54210085288.